pkgs/by-name/ve/veroroute/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  qtbase,
+  wrapQtAppsHook,
+  qmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "veroroute";
+  version = "2.39";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/veroroute/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

Download link should continue to work when pname is overridden

```suggestion
    url = "mirror://sourceforge/veroroute/veroroute-${finalAttrs.version}.tar.gz";
```

---

Done.